src/geometry/manifold/algebra/smooth_functions.lean

```diff
@@ -80,6 +85,11 @@ instance smooth_map_group {G : Type*} [group G] [topological_space G]
   mul_left_inv := λ a, by ext; exact mul_left_inv _,
   .. smooth_map_monoid }
 
+@[simp, to_additive]
+lemma smooth_map.coe_inv {G : Type*} [group G] [topological_space G]
+  [charted_space H' G] [lie_group I' G] (f : C^∞⟮I, N; I', G⟯) :
+  ⇑f⁻¹ = f⁻¹ := rfl
```

Comment:
Can you add `coe_div` too, primarily for the `coe_sub` that generates?

Reply:
Also yes and done.